<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterForm</class>
 <widget class="QWidget" name="FilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Packet Filtering</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sent</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="LE_SEND_NEW_PACKET"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="CB_SEND_FILTERED"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="CB_SEND_FILTERING_TYPE">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Packet name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Contains</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ends with</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="PB_SEND_FILTER">
       <property name="text">
        <string>Add to filter</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="CB_SEND_FILTER">
       <property name="text">
        <string>Filter ?</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="PB_SEND_DELETE">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <item row="0" column="1">
      <widget class="QComboBox" name="CB_RCVD_FILTERED"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="LE_RCVD_NEW_PACKET"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="CB_RCVD_FILTERING_TYPE">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Packet name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Contains</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ends with</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rcvd</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="PB_RCVD_FILTER">
       <property name="text">
        <string>Add to filter</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="CB_RCVD_FILTER">
       <property name="text">
        <string>Filter ?</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="PB_RCVD_DELETE">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
